Gemfile

```diff
@@ -11,7 +11,7 @@ gem 'concurrent-ruby-edge', Concurrent::EDGE_VERSION, options
 gem 'concurrent-ruby-ext', Concurrent::VERSION, options.merge(platform: :mri)
 
 group :development do
-  gem 'rake', '~> 12.0'
+  gem 'rake', (RUBY_VERSION < '2.2.0') ? '~> 12.0' : '~> 13.0'
```

Comment:
There is a utility file for this which can be safely loaded here `lib/concurrent-ruby/concurrent/utility/engine.rb`, then `Concurrent.ruby_version :<, 2, 2, 0` can be used. It does proper version comparison.

Reply:
Thanks!